mavros_extras/scripts/mavteleop

```diff
@@ -77,6 +99,8 @@ rc_channels = {
     'throttle': RCChan('throttle', 2, 0.0)
 }
 
+rc = OverrideRCIn() # must be global to retain mode value
```

Comment:
I'm not a fan of global variables where we can avoid it.

Store required value in RCMode class, anyway you apply that on each turn.